### ISSUE TITLE
Accessibility: Odd ListTile/DropdownButton accessibility interaction under Android and TalkBack

### ISSUE THREAD

**Issue by ndarilek** [flutter/flutter#36967]
I'm an Android TalkBack user, and have encountered an odd case where a `ListTile` with a trailing `DropdownButton` doesn't allow for interaction with the button. That is, I can't double-tap the list item and interact with the button, nor does the `DropdownButton` appear to be separate from the `ListTile` title.

I'm fairly new to Flutter and am probably doing something wrong here, particularly since my buttons aren't speaking any selected values. But I've encountered this behavior [here](https://gitlab.com/redsolver/preferences/blob/master/lib/dropdown_preference.dart) as well, so either we're both doing something wrong, or we've hit an accessibility edge case.

My expectations are two-fold. I should be able to touch the `ListTile` title as a separate item, including tapping on it to trigger its action, and I should be able to interact with the trailing `DropdownButton`.

Thanks for any help. Here is my reproduction:

```
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'ListViews',
      home: Scaffold(
        appBar: AppBar(title: Text('ListViews')),
        body: ListView.builder(
          itemCount: 10,
          itemBuilder: (context, index) {
            return ListTile(
              title: Text("Row ${index}"),
              trailing: DropdownButton(
                items: [
                  DropdownMenuItem(
                    value: "first",
                    child: Text("First"),
                  ),
                  DropdownMenuItem(
                    value: "second",
                    child: Text("Second"),
                  ),
                ],
                value: "first",
              ),
            );
          },
        ),
      ),
    );
  }
}
```

**Comment by perclasson** [comment]
What version of flutter are you running (`flutter doctor -v`)?

I am not able to reproduce this on TalkBack or VoiceOver. It is working as expected for me:
![listtile dropdown talkback](https://user-images.githubusercontent.com/1770678/72473450-b5aee380-37b4-11ea-8588-68eb07f384c1.gif) I'll close this ticket for now, feel free to reopen if you have more details.
